spec/sources/buildkite_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'spec_helper'
+require 'danger/ci_source/circle'
+
+describe Danger::CISource::Buildkite do
+  it 'validates when buildkite env var is found' do
+    env = { "BUILDKITE" => "true" }
+    expect(Danger::CISource::Buildkite.validates?(env)).to be true
+  end
+
+  it 'doesnt validate when buildkite is not found' do
+    env = { "HAS_JOSH_K_SEAL_OF_APPROVAL" => "true" }
+    expect(Danger::CISource::Buildkite.validates?(env)).to be false
+  end
+
+  it 'gets out a repo slug from a git+ssh repo and pull request number' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.